1.data_preprocessing/2.format_bulk.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>It may be useful to create some plots of the data or summary statistics to make sure you get a good understanding/feeling for the data before you process it further. You don't have to do it here, but I think it's always good to get acquainted with the raw data a bit (like a PCA colored by race / other metadata values)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>